vavr-benchmark/build.gradle

```diff
@@ -0,0 +1,15 @@
+
+description = 'Vavr Benchmark'
+dependencies {
+  compile project(':vavr')
+    compile group: 'org.scalaz', name: 'scalaz-core_2.12', version:'7.3.0-M8'
+    compile group: 'org.clojure', name: 'clojure', version:'1.9.0-alpha14'
+    compile group: 'org.pcollections', name: 'pcollections', version:'2.1.2'
+    compile group: 'org.functionaljava', name: 'functionaljava-java8', version:'4.6'
+    compile group: 'org.eclipse.collections', name: 'eclipse-collections', version:'8.0.0'
+    compile group: 'io.usethesource', name: 'capsule', version:'0.6.1'
+    compile group: 'org.openjdk.jol', name: 'jol-core', version:'0.7.1'
+    compile group: 'org.jetbrains', name: 'annotations', version:'15.0'
+    testCompile group: 'junit', name: 'junit', version:'4.12'
```

Comment:
It may be appropriate to keep all versions in a common `gradle.properties` file. At least for `junit` and `assertj`